fdbserver/DataDistribution.actor.cpp

```diff
@@ -4272,24 +4338,22 @@ ACTOR Future<Void> perpetualStorageWiggler(AsyncVar<bool>* stopSignal,
 				           "PerpetualStorageWigglePause",
 				           self->distributorId)
 				    .detail("Primary", self->primary)
-				    .detail("ProcessId", pid)
+				    .detail("ProcessId", id)
 				    .detail("BestTeamKeepStuckCount", self->bestTeamKeepStuckCount)
 				    .detail("ExtraHealthyTeamCount", extraTeamCount)
-				    .detail("HealthyTeamCount", self->healthyTeamCount)
-				    .detail("StorageCount", movingCount);
+				    .detail("HealthyTeamCount", self->healthyTeamCount);
```

Comment:
the re-included machine should only accept moving to it for healing unhealthy teams

Reply:
@sfc-gh-jslocum Question:  When the cluster becomes unhealthy, the perpetual wiggler will re-include the process that is currently being wiggled and go idle.  Even though DD will prioritize data moves required to heal the cluster, I think it's still possible load balancing would start moving data _back to_ the process that was being wiggled, which would be on average half full at that point.  Is this correct?  Ideally we would not want this to happen, but we would allow data moves to the process that was being wiggled if the moves are to heal the cluster.

Reply:
Yes that's correct. The wiggle priority of moves is lower than the healing priority.
My understanding was that pauses *should* be only in the case where the cluster is pretty unhealthy, and we need this host back in the cluster to help heal, so moving stuff back to that node and later restarting the wiggle from scratch for that process was acceptable.
In fact the data moves would likely prioritize moving *to* the former wiggling host, and we select destination teams mostly on disk bytes available, and it would necessarily have more available (assuming homogenous hardware).

If this behavior is not desired, I think you could add logic to prefer moving to teams that aren't wiggling fairly easily though.

Reply:
I created a followup issue for this:
https://github.com/apple/foundationdb/issues/6325